subprojects/docs/src/samples/testing/junitplatform/mix/groovy/build.gradle

```diff
@@ -29,7 +29,3 @@ dependencies {
     testRuntimeOnly 'org.junit.vintage:junit-vintage-engine:5.1.0'
```

Comment:
Can you sanity check the dependencies in our samples?  It looks like we could upgrade to a newer version of junit-jupiter and make the sample simpler.